client/src/models/imageScrubber.js

```diff
@@ -1,113 +1,107 @@
-        })
+    async getTreesWithImages({
+      page,
+      rowsPerPage,
+      orderBy = "id",
+      order = "asc"
+    }) {
+      const query = `${API_ROOT}/trees?filter[order]=${orderBy} ${order}&filter[limit]=${rowsPerPage}&filter[skip]=${page *
+        rowsPerPage}&filter[fields][imageUrl]=true&filter[fields][lat]=true&filter[fields][lon]=true&filter[fields][id]=true&filter[fields][timeCreated]=true&filter[fields][timeUpdated]=true&filter[where][active]=true&field[imageURL]`;
+      Axios.get(query).then(response => {
+        this.getTrees(response.data, {
+          page: page,
+          rowsPerPage: rowsPerPage,
+          orderBy: orderBy,
+          order: order
+        });
+      });
     },
     async getLocationName(payload, rootState) {
-      if ((rootState.trees.byId[payload.id] &&
-        rootState.trees.byId[payload.id].location &&
-        rootState.trees.byId[payload.id].location.lat !== payload.lat &&
-        rootState.trees.byId[payload.id].location.lon !== payload.lon) ||
-        (
-          !rootState.trees.byId[payload.id] || !rootState.trees.byId[payload.id].location
-        )) {
-        const query = `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${payload.latitude}&lon=${payload.longitude}`
-        Axios.get(query)
-          .then((response) => {
-            this.receiveLocation(response.data, payload)
-          })
+      if (
+        (rootState.trees.byId[payload.id] &&
+          rootState.trees.byId[payload.id].location &&
+          rootState.trees.byId[payload.id].location.lat !== payload.lat &&
+          rootState.trees.byId[payload.id].location.lon !== payload.lon) ||
+        (!rootState.trees.byId[payload.id] ||
+          !rootState.trees.byId[payload.id].location)
+      ) {
+        const query = `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${
+          payload.latitude
+        }&lon=${payload.longitude}`;
+        Axios.get(query).then(response => {
+          this.receiveLocation(response.data, payload);
+        });
       } else {
-        this.receiveLocation(null, { id: payload.id, address: 'cached' })
+        this.receiveLocation(null, { id: payload.id, address: "cached" });
       }
     },
     async toggleTreeActive(id, isActive) {
       const query = `${API_ROOT}/trees/${id}/`;
-      const data = { "active": isActive };
+      const data = { active: isActive };
       Axios.patch(query, data)
-        .then((res) => {
+        .then(res => {
           if (res.status === 200) {
-            this.receiveStatus(res.data)
+            this.receiveStatus(res.data);
           }
         })
-        .catch(err => console.error(`ERROR: FAILED TO RETRIEVE STATUS ${err}`))
-    },
+        .catch(err => console.error(`ERROR: FAILED TO RETRIEVE STATUS ${err}`));
+    }
   }
-}
+};
 
-export default imageScrubber
+export default imageScrubber;
```

Comment:
not sure what happened with the merge, but this was moved and is no longer being used. we can remove it later

Reply:
this preview is not so useful, if interested, see the full changes